R/lockbox.R

```diff
@@ -132,8 +132,12 @@ lockbox_download_dir <- function() {
 
 #' The transient lockbox library path.
 lockbox_transient_dir <- function() {
-  getOption("lockbox.transient_dir") %||%
+  if (!is.null(.lockbox_env$session_id)) {
```

Comment:
Nitpicking question; isn't this breaking the pattern a little by forcing a reassignment of the option on every function call when .lockbox_env$session_id is not NULL?  Couldn't the infix operator be followed by an ifelse? Maybe there is something else you're trying to accomplish here that I'm not following. :(